test/test_game.py

```diff
@@ -838,6 +838,32 @@ def test_penalization_after_invalid_moves(
         self.assertEqual(game.current_player.score, expected_score)
         self.assertEqual(game.game_is_active, game_active)
 
+    @parameterized.expand([
+        (PLAYER_1, 10, PLAYER_2, 9),
+        (PLAYER_1, 200, PLAYER_2, 199),
+        (PLAYER_2, 10, PLAYER_1, 9),
+        (PLAYER_2, 15, PLAYER_1, 14),
+    ])
+    def test_next_turn(self, current_player, current_remaining_moves,
```

Comment:
I would use `initial_player` and `initial_remaining_moves` here to avoid confusion in the test.

Reply:
Exellent, I didn't know what to call them jaja. Thank you, my shadow master